bundles/org.openhab.binding.jeelink/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -534,4 +566,11 @@
 		<label>@text/channel-type.power-frequency.label</label>
 		<description>@text/channel-type.power-frequency.description</description>
 	</channel-type>
+
+	<channel-type id="voltage-current" advanced="true">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage</label>
+		<description>Voltage</description>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
```

Comment:
This channel is added, but it does not seem to be used by the new Thing. I guess this should be removed as the thing allready has two (both potential and current) related and existing channels